package/yast2-apparmor.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-apparmor
-Version:        3.2.1
+Version:        3.2.1.1
```

Comment:
See the comment above, https://github.com/yast/yast-apparmor/pull/41#pullrequestreview-263046884